openlibrary/templates/books/edit.html

```diff
@@ -93,6 +93,10 @@ <h1>$:title</h1>
             <div class="label">
                 <label for="author-$0">$_('Author')</label>
                 <span class="tip">$:_('You can search by author name (like <em>j k rowling</em>) or by Open Library ID (like <a href="/authors/OL23919A" target="_blank"><em>OL23919A</em></a>).')</span>
+                <br/>
+                <noscript>
+                    <div class="flash-messages"><p>Author editing requires javascript</p></div>
```

Comment:
This message needs to be internationalized

Reply:
@tfmorris and how exactly can that be done, sir?

Reply:
Same way as above using `$:_`